### DISCUSSION TITLE
adding stuff to the client...?

### DISCUSSION THREAD

**Discussion by daguy2023** [Fedoraware/Fedoraware#1114] [Ideas]
Is it possible to add a new hack by going to ./src/features/ and making a folder with necessary files?
more importantly, how hard would it be to add a new feature?

**Comment by ghost** [comment]
In future I think you can create discussions under the questions sub category instead of ideas.

To answer your question, you can add feature pretty easily with this cheat, as long as you have an understanding of c++, as well as how to make that feature in TF2, you can simply make a new folder in Features/ make your files and then implement them where necessary, if what you are doing is a small feature it might be more fitting if it is added to Misc.cpp and Misc.h. If you are interested in creating features for fedoraware and want any assistance on how the cheat already does things you should join the discord and ask for access to the contributors channel, where you can then ask for help.
